hosts/dailymotion/mexp-ev-dailymotion-service.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+// Made to work with External Videos functions.
+// If we have no authors let's get out of here.
+$options = get_option( 'sp_external_videos_options' );
+if( empty( $options['hosts']['dailymotion']['authors'] ) ) return;
+
+// Classname should not conflict with default MEXP Services
+// in case MEXP is already installed
+class MEXP_EV_Dailymotion_Service extends MEXP_Service {
+
+  const SERVICE = 'dailymotion';
+  const EV_SERVICE = 'ev_dailymotion';
+
+	public function __construct() {
+
+    require_once dirname( __FILE__ ) . '/mexp-ev-dailymotion-template.php';
+		# Go!
+		$this->set_template( new MEXP_EV_Dailymotion_Template );
+	}
+
+	public function load() {
+
+		add_action( 'mexp_enqueue', array( $this, 'enqueue_statics' ) );
+		add_filter( 'mexp_tabs', array( $this, 'tabs' ), 10, 1 );
+		add_filter( 'mexp_labels', array( $this, 'labels' ), 10, 1 );
+
+	}
+
+	public function enqueue_statics() {
+
+		$mexp = Media_Explorer::init();
+
+	}
+
+  public function request( array $request ) {
+
+    $params = $request['params'];
+    $author_id = $params['author_id'];
+    $search = $params['q'];
+
+    $args = array(
+      'post_type' => 'external-videos',
+      'nopaging' => true,
+      'meta_query' => array(
+        array(
+          'key' => 'host_id',
+          'value' => self::SERVICE
+        ),
+        array(
+          'key' => 'author_id',
+          'value' => $author_id
+        )
+      )
+    );
+    if( $search ) $args['s'] = $search;
+
+    // error_log( '$args: ' . print_r( $args, true) );
+
+    $query_response = get_posts( $args );
+
+    // error_log( '$query_response: ' . print_r( $query_response, true) );
+
+		// Create the response for the API
+		$response = new MEXP_Response();
+
+		if ( !isset( $query_response ) )
+			return false;
+
+    // Set up items
+		foreach ( $query_response as $index => $video ) {
+      setup_postdata( $video );
```

Comment:
Yes this is weird! I have my editor set to do double spaces, no tabs, and everything seemed ok at commit time.